packages/api/lib/deleter.js

```diff
@@ -15,7 +15,7 @@
 
 const firebaseEncode = require('../lib/firebase-encode');
 
-async function deleteRepo (client, repoId) {
+async function deleteRepo (repoId, client) {
```

Comment:
@bcoe, @cedpeters. The "flaky" test was actually deterministic :):
That line changed the signature of `deleteRepo`. But `deleteRepo` was used in `packages/api/test/get-batches.js`.
[Updating the signature](https://github.com/GoogleCloudPlatform/flaky-service/commit/e3f36970ea6e7646c2edea0d0d920538f8338b3a#diff-6b9db70b6a85df0db02734e6f54b81e0R365) fixed the test: